dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/message/MessageImpl.java

```diff
@@ -1087,7 +982,11 @@ public boolean containsValue(Object value) {
         }
         
         for (Object o : propertyValues) {
-            if (o.equals(value)) {
+            if (o == null) {
+                if (value == null) {
+                    return true;
+                }
+            } else if (o != NOT_FOUND && o.equals(value)) {
```

Comment:
You might want to change this a little.  If an equals method doesn't handle null it could get messy.
`if (value == null) {
    for (Object o : propertyValue) {
        if (o == null) {
           return true;
        }
    }
} else {
    for (Object o : propertyValues) {
        if (o != null && o != NOT_FOUND && value.equals(o) {
            return true;
        }
    }
}`

This prevents passing a null to o.equals and if value is null and it calls equals on the value passed in instead of the one in the propertyValues. We do not know the impl of those equals where the equals of the value passed in needs to work correctly by definition.